### DISCUSSION TITLE
How can I ignore unhandled errors?

### DISCUSSION THREAD

**Discussion by leonheess** [vitest-dev/vitest#1640] [Q&A]
see title

**Comment by Demivan** [comment]
You can use try...catch.

Or do you want to enable this globally? I would not recommend that.

**Reply by leonheess** [reply]
Globally, you can find more details here: https://stackoverflow.com/questions/72983575/how-to-mock-relative-requests-in-vitest

**Reply by Demivan** [reply]
Can you share a reproduction?

By the sound of it, it will be easier for you to mock fetch module and not the network request.
